MidasCivil_Adapter/Convert/ToMidasCivil/Properties/FromMetaData.cs

```diff
@@ -0,0 +1,92 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Text;
+using BH.oM.Adapters.MidasCivil;
+using System.Collections.Generic;
+
+namespace BH.Adapter.Adapters.MidasCivil
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static StringBuilder FromMetadata(Metadata metadata)
+        {
+            StringBuilder midasMetadata = new StringBuilder();
+            List<Review> reviews = metadata.Reviews;
+            midasMetadata.AppendLine("*PROJINFO");
+            midasMetadata.AppendLine($"PROJECT={metadata.ProjectNumber}");
+            midasMetadata.AppendLine($";DESIGNSTAGE={metadata.DesignStage}");
+            midasMetadata.AppendLine($";PROJECTLEAD={metadata.ProjectLead}");
+            midasMetadata.AppendLine($"REVISION={metadata.Revision}");
+            midasMetadata.AppendLine($"USER={metadata.Author}");
+            midasMetadata.AppendLine($"EMAIL={metadata.Email}");
+            midasMetadata.AppendLine($"ADDRESS={metadata.Location.Replace(System.Environment.NewLine, ", ")}");
+            midasMetadata.AppendLine($"CLIENT={metadata.Client}");
+            midasMetadata.AppendLine($"TITLE={metadata.ProjectName}");
+            midasMetadata.AppendLine($"ENGINEER={metadata.Author}");
+            midasMetadata.AppendLine($"EDATE={metadata.CreationDate.ToString("yyyy-MM-dd")}");
+            midasMetadata.AppendLine($";DESCRIPTION={metadata.Description}");
+            midasMetadata.AppendLine($";DISCIPLINE={metadata.Discipline}");
+
+            if(reviews.Count > 4)
+            {
+                reviews.RemoveRange(1, reviews.Count - 2);
+                Engine.Reflection.Compute.RecordWarning("The number of reviews exceeds the limit of MidasCivil. Only the first, last and second to last reviews will be recorded.");
+            }
+            if(reviews.Count >= 1)
+            {
+                midasMetadata.AppendLine($"CHECK1={reviews[0].Reviewer}");
+                midasMetadata.AppendLine($"CDATE1={reviews[0].ReviewDate.ToString("yyyy-MM-dd")}");
+            }
+            if (reviews.Count >= 2)
+            {
+                midasMetadata.AppendLine($"CHECK2={reviews[1].Reviewer}");
+                midasMetadata.AppendLine($"CDATE2={reviews[1].ReviewDate.ToString("yyyy-MM-dd")}");
+                Engine.Reflection.Compute.RecordWarning("Only comments from the last review will be pushed to MidasCivil.");
```

Comment:
Should this warning be included in all instances where reviews.Count > 1?

Reply:
it is included because the statement is an **If** not **Else If**.